docs/source/getting-started/index.rst

```diff
@@ -1,9 +1,9 @@
-Getting Started
-===============
+Getting Started as a Code User 
+=========================
```

Comment:
```suggestion
Getting Started
==================
```
I'm not sure this needs changing as a getting started section is a very common section across different projects' documentation. The confusion might come from not having enough direction to this section on the front page.